src/Checkout.css

```diff
@@ -0,0 +1,22 @@
+.checkout {
+  display: flex;
+  padding: 20px;
+  background-color: white;
```

Comment:
use variables or hex code for colors

Reply:
for more that 10px you can use rem convention